tests/integration_tests.rs

```diff
@@ -1,61 +1,144 @@
+            FOREIGN KEY (member_id) REFERENCES member(id)
+        )"#,
+        r#"
+        CREATE TABLE IF NOT EXISTS leetcode_stats (
+            id SERIAL PRIMARY KEY,
+            member_id INT UNIQUE NOT NULL,
+            leetcode_username VARCHAR(255) NOT NULL,
+            problems_solved INT NOT NULL,
+            easy_solved INT NOT NULL,
+            medium_solved INT NOT NULL,
+            hard_solved INT NOT NULL,
+            contests_participated INT NOT NULL,
+            best_rank INT NOT NULL,
+            total_contests INT NOT NULL,
+            FOREIGN KEY (member_id) REFERENCES member(id)
+        )"#,
+        r#"
+        CREATE TABLE IF NOT EXISTS codeforces_stats (
+            id SERIAL PRIMARY KEY,
+            member_id INT UNIQUE NOT NULL,
+            codeforces_handle VARCHAR(255) NOT NULL,
+            codeforces_rating INT NOT NULL,
+            max_rating INT NOT NULL,
+            contests_participated INT NOT NULL,
+            FOREIGN KEY (member_id) REFERENCES member(id)
+        )"#,
+    ];
+
+    for query in queries {
+        sqlx::query(query)
+            .execute(&pool)
+            .await
+            .expect("Failed to execute query");
+    }
+    pool
 }
 
 // Helper function to clean up test data
+
 async fn cleanup_test_data(pool: &PgPool) {
-    // sqlx::query("DELETE FROM leaderboard")
-    //     .execute(pool)
-    //     .await
-    //     .unwrap();
-    // sqlx::query("DELETE FROM leetcode_stats")
-    //     .execute(pool)
-    //     .await
-    //     .unwrap();
-    // sqlx::query("DELETE FROM codeforces_stats")
-    //     .execute(pool)
-    //     .await
-    //     .unwrap();
-    // sqlx::query("DELETE FROM Member")
-    //     .execute(pool)
-    //     .await
-    //     .unwrap();
+    print!("called");
+    let cleanup_query = r#"
+        DO $$
+        DECLARE
+            seq RECORD;
+        BEGIN
+            -- Truncate tables if they exist
+            BEGIN
+                TRUNCATE TABLE leaderboard, leetcode_stats, codeforces_stats, member RESTART IDENTITY CASCADE;
+            EXCEPTION
+                WHEN undefined_table THEN
+                    -- Ignore errors if tables don't exist
+                    RAISE NOTICE 'Tables do not exist, skipping TRUNCATE.';
+            END;
+
+            -- Reset sequences only if they exist
+            FOR seq IN
+                SELECT c.relname
+                FROM pg_class c
+                JOIN pg_namespace n ON n.oid = c.relnamespace
+                WHERE c.relkind = 'S' AND n.nspname = 'public'
+            LOOP
+                BEGIN
+                    EXECUTE 'ALTER SEQUENCE ' || seq.relname || ' RESTART WITH 1';
+                EXCEPTION
+                    WHEN undefined_object THEN
+                        -- Ignore errors if sequences don't exist
+                        RAISE NOTICE 'Sequence % does not exist, skipping.', seq.relname;
+                END;
+            END LOOP;
```

Comment:
Improve the comments for this, currently its very hard to tell what the purpose of this query is.